lib/browser/api/extensions.js

```diff
@@ -183,6 +183,15 @@ var updateWindow = function (windowId, updateInfo) {
   }
 }
 
+var removeWindow = function (windowId) {
+  let win = BrowserWindow.fromId(windowId)
+  if (win) {
+    win.close()
+  } else {
+    console.warn('chrome.windows.remove could not find windowId ' + windowId)
```

Comment:
it isn't clear from the docs, but this method should return an error if the window id doesn't exist. See tabs.remove in tabs_bindings.js for an example